samples/openapi3/client/petstore/dart-dio-next/petstore_client_lib_fake_tests/test/api/fake_api_test.dart

```diff
@@ -36,7 +36,7 @@ void main() {
             'int64': '9223372036854775807',
             'date': '2020-08-11',
             'dateTime': '2020-08-11T12:30:55.123Z',
-            'binary': "Instance of 'MultipartFile'",
+            'binary': '[0, 1, 2, 3, 4, 5]',
```

Comment:
Not sure this makes sense to send over the wire like this, but better than before in any case.